docs/1___overview.adoc

```diff
@@ -67,8 +67,12 @@ The interface types have large parts in common, defined in <<fmi-common-concepts
 In particular:
 
 * FMI Application Programming Interface \(C) -- <<general-mechanisms>> +
-All required equations or tool coupling computations are evaluated by calling standardized C functions.
+All required equations or tool coupling computations are evaluated by calling standardized C functions from the importer into the FMU.
+The FMU can signal certain events back to the importer using callback functions provided by the importer.
 C is used because it is the most portable programming language today and is the only programming language that can be utilized in all embedded control systems.
+The FMI API does not restrict what operating system services an FMU can use on the platform it runs on.
+However, for maximal portability, any dependency on the target platform should be minimized and operating system services should be accessed only through standard libraries.
+Special run-time requirements should be documented in the appropriate directory inside the <<fmu-distribution,ZIP file>>.
```

Comment:
I think this is very important for each vendor investing into FMU support as they are aiming for high compatibility with other tools.
* Can we emphasize it a little more? Maybe "... it is recommend..." or similarly?
* I am not sure if "run-time requirements" is too vague. What are you thinking of? Does this include dependencies on certain linux packages as well?

Reply:
- "should" is stronger than "is recommended" (See RFC 2119), if I remember correctly.
- "run-time requirements" must be vague, I think: We have no way of listing all potential dependencies that might or might not be available from one platform to the next, especially not for the future. Most of this is truely academic, because no one in their right mind would introduce a run-time dependency without need. What we need to say here is that the standard is silent on this issue and leaves this to the care of the FMU exporter to document "what is needed".